pkg/controllers/deprovisioning/expiration.go

```diff
@@ -67,27 +66,15 @@ func (e *Expiration) ShouldDeprovision(ctx context.Context, c *Candidate) bool {
 	return c.Node.Annotations[v1alpha5.VoluntaryDisruptionAnnotationKey] == v1alpha5.VoluntaryDisruptionExpiredAnnotationValue
 }
 
-// SortCandidates orders expired nodes by when they've expired
-func (e *Expiration) filterAndSortCandidates(ctx context.Context, nodes []*Candidate) ([]*Candidate, error) {
-	candidates, err := filterCandidates(ctx, e.kubeClient, e.recorder, nodes)
-	if err != nil {
-		return nil, fmt.Errorf("filtering candidates, %w", err)
-	}
-	sort.Slice(candidates, func(i int, j int) bool {
-		return node.GetExpirationTime(candidates[i].Node, candidates[i].provisioner).Before(node.GetExpirationTime(candidates[j].Node, candidates[j].provisioner))
-	})
-	return candidates, nil
-}
-
 // ComputeCommand generates a deprovisioning command given deprovisionable nodes
 func (e *Expiration) ComputeCommand(ctx context.Context, nodes ...*Candidate) (Command, error) {
```

Comment:
```suggestion
func (e *Expiration) ComputeCommand(ctx context.Context, candidates ...*Candidate) (Command, error) {
```

Maybe rename this to candidates at the top-level and just call it candidates everywhere?